src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(Color color, double radius) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create separate variables for 'color' and 'radius' and pass them on for better code readability.